src/components/BudgetTable/BudgetTable.tsx

```diff
@@ -66,9 +79,20 @@ const BudgetTable: React.FC<BudgetTableProps> = ({ rows, itemsPerPage }) => {
           <tbody>
             {paginateTable().map((row, index) => (
               <tr className="hover:bg-slate-50" key={index}>
-                {Object.values(row).map((value, index) => (
-                  <td className="text-start border p-2" key={index}>
-                    {value}
+                {headers.map((header) => (
+                  <td className="text-start border p-2" key={header}>
+                    <div className="flex gap-4 items-center">
+                      {header === "name" && (
```

Comment:
with dynamic rendering, create a component for this bullet, to improve style and animation